examples/sandbox/src/pages/ecommerce/styles.scss

```diff
@@ -0,0 +1,31 @@
+@tailwind base;
```

Comment:
why do we need to change to sass from vanilla css here?

Reply:
Previously `styles.css` applied styles globally to the search-ui classes and affected other examples. So I added a wrapper `ecommerce-container` to apply styles only for this example. To avoid repeating that logic in plain css, I rewrote it using scss.